src/main/java/com/cqse/teamscaleupload/TeamscaleUpload.java

```diff
@@ -168,11 +171,11 @@ private static Input parseArguments(String[] args) {
         parser.epilog("INPUTFILE" +
                 "\n" +
                 "\n" +
-                "The file you provide via --input consists of file patterns in the same format as used on the command line" +
-                " and optionally sections that specify additional report file formats." +
-                " The entries in the file are separated by line breaks. Uses the format specified with --format," +
-                " unless you overwrite the format explicitly for a set of patterns:\n\n" +
-                "pattern1/**.simple\n" +
+                "The input file allows to upload report files for different file patterns in one upload session." +
+                " Each section of reports must start with a specification of the report format. The report file" +
+                " patterns have the same format as used on the command line." +
+                " The entries in the file are separated by line breaks.\n\n" +
+                "Example:\n\n" +
```

Comment:
can you add another section with a different format to the example please? just to make it explicit that you can use the format header multiple times.

Reply:
I do not quite understand what you mean. The example already contains multiple formats (`[jacoco]` and `[findbugs]`).

Or do you mean something else?

Reply:
Aaah sorry, the GitHub Diff had collapsed the rest of the example. My bad, sorry! :-)